examples/hello_zig/Makefile

```diff
@@ -23,6 +23,7 @@ include $(APPDIR)/Make.defs
 # Hello, Zig! Example
 
 MAINSRC = hello_zig_main.zig
+ZIGFLAGS += -fsingle-threaded
```

Comment:
should it be a common flags?

Reply:
Sorry, this flag is only needed when for example trying to make it compatible with rust_app which requires lib to be single-thread.